mlos_bench/mlos_bench/storage/base_storage.py

```diff
@@ -83,7 +83,8 @@ def experiment(self, *,
                    trial_id: int,
                    root_env_config: str,
                    description: str,
-                   opt_target: str) -> 'Storage.Experiment':
+                   opt_target: str,
+                   opt_direction: str) -> 'Storage.Experiment':
```

Comment:
I think that optimization direction can be nullable - e.g. when we run a standalone benchmark. (we should still have the optimization target, though, because even in a standalone benchmark we still measure something)